packages/patrol_cli/lib/src/test_bundler.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:file/file.dart';
+import 'package:meta/meta.dart';
+
+class TestBundler {
+  TestBundler({required Directory projectRoot})
+      : _projectRoot = projectRoot,
+        _fs = projectRoot.fileSystem;
+
+  final Directory _projectRoot;
+  final FileSystem _fs;
+
+  File createTestBundle(List<String> testFilePaths) {
+    if (testFilePaths.isEmpty) {
+      throw ArgumentError('testFilePaths must not be empty');
+    }
+
+    final contents = '''
+// ignore_for_file: invalid_use_of_internal_member,
+// depend_on_referenced_packages, directives_ordering
+
+import 'dart:async';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:patrol/patrol.dart';
+import 'package:patrol/src/native/contracts/contracts.pbgrpc.dart';
+import 'package:test_api/src/backend/invoker.dart';
+
+// START: GENERATED CODE
+${generateImports(testFilePaths)}
+// END: GENERATED CODE
+
+Future<void> main() async {
+  // TODO: Create and use PatrolNativeTestService instead of NativeAutomator
+  final nativeAutomator = NativeAutomator(config: NativeAutomatorConfig());
+  final binding = PatrolBinding.ensureInitialized();
+  // Create a PatrolAppService.
```

Comment:
I like this comment, but it has quite a few typos and its structure could be improved.